mongodb-odbc-driver/bin/run-atlas-integration-tests.sh

```diff
@@ -38,8 +38,13 @@
 
     elif [ "macos" = "$PLATFORM" ]; then
         if ! hash iodbctestw &> /dev/null; then
-            echo 'building iODBC'
-            "$SCRIPT_DIR"/build-iodbc.sh
+            echo 'downloading iodbctestw'
+            mkdir -p "$BUILD_DIR"
```

Comment:
[nit] This should have  been in a new download-iodbc.sh script to follow the convention.

Reply:
The convention was really "is this a lot of code? (I know because I wrote it haha)", but I did this anyway